docs/manifest_by_file.md

```diff
@@ -177,7 +177,10 @@ File Path | Manifest
 /var/log/dpkg\* | diagnostic, eg, normal 
 /var/log/evtcollforw\*.log | site-recovery 
 /var/log/hadoop-yarn/yarn/\*.log | hdinsight 
-/var/log/hive/hivemetastore.log | hdinsight 
+/var/log/hdinsight-agent/hdinsight-agent.\* | hdinsight 
+/var/log/hdinsight-provisioning-agent/hdinsight-provisioning-agent.log | hdinsight 

```

Comment:
should we then do wild card for this as well?

Reply:
These two md files are generated( see readme). their content should match hdinsight manifest file. so the md files would not have wild card if the original hdinsight file doesn't have them. Also what you have in hdinsight is good in term of not having * wild card when you know the exact file path/names. This will enable us to treat each file differently in  credscan process, for example, to put "noscan" at the end of binary file or files that for sure would not contains secrets. All this help improve the performance of IID service, which benefits everyone. If you already know that a file is binary or a file without secret, please put "noscan" at the end ( see examples in other manifest files for syntax)